scripts/GSAS-II/install_gsas_latest.bat

```diff
@@ -0,0 +1,17 @@
+@echo off
+@echo Running install_gsas_proxy.bat
+
+@set DEV_PYTHON_EXE=%~dp0..\..\external\src\ThirdParty\lib\python2.7\python.exe
+@set RELEASE_PYTHON_EXE=%~dp0..\..\bin\python.exe
+
+if exist %DEV_PYTHON_EXE% (
+   @echo Using %DEV_PYTHON_EXE%
+   %DEV_PYTHON_EXE% install_gsas_proxy.py
+) else (if exist %RELEASE_PYTHON_EXE% (
+  @echo Using %RELEASE_PYTHON_EXE%
+   %RELEASE_PYTHON_EXE% install_gsas_proxy.py
+) else (
+   echo Could not find Mantid Python executable
+)
+)
+pause
```

Comment:
The script looks good. I can see that the `pause` is necessary so that users can see the script has finished but for automation we'll need to be able to run this in a non-interactive manner. How about adding a flag for the script, something like `--non-interactive` that when passed to the script it will skip the `pause` part?